pkg/sql/lease.go

```diff
@@ -1734,7 +1734,10 @@ func (m *LeaseManager) isDraining() bool {
 
 // SetDraining (when called with 'true') removes all inactive leases. Any leases
 // that are active will be removed once the lease's reference count drops to 0.
-func (m *LeaseManager) SetDraining(drain bool) {
+//
+// The reported callback, if non-nil, is called to report the number of
```

Comment:
These descriptions seem ambiguous about whether it's counting the "work that needed to be done and which may or may not have been done by the time this call returns" or "the work that was actually done". Looks like it's the former, at least here (and I assume consistently throughout).